docs/Policy/Maintenance/Scenario_Maintenance_Policy.md

```diff
@@ -152,38 +161,45 @@ And we wanted to deprecate testing `ACM2.5` then we will just remove the followi
     - chain: interop-acm
     workflow: ipi-aws
 ```
+
 While scenario expansions must have a passing rehearsal job for the PR to be merged, a deprecation does not.
 
 ## OpenShift Deployment
+
 >This responsibility is abstracted away from both the CSPI-QE team and PQE teams to the workflow OWNERS. 
 
 This responsibility falls on the owner of the workflow that is being used to deploy the OpenShift environment needed by the OpenShift Layered Product Scenario. Each workflow will most likely have a different owner. If you need support for a problem in a workflow you'll need to find the workflow that you are using in the [release repo's step-registry](https://github.com/openshift/release/tree/master/ci-operator/step-registry) and contact the users in the OWNERS file.
 
 ## Layered Product Deployment
+
 >This is the responsibility of the PQE team.
 
 The model being followed by this [layered product onboarding](../../Onboarding/Onboarding_Guide.md) is built on the idea that automation that is already created and being maintained should not be recreated by a different team attempting to achieve the same result. Therefore the onboarding of the scenario will ensure that the layered product deployment relies on automation built by that product's QE team. If there are failures in the OCP CI scenario for the layered product deployment then it will need to be fixed at the source, which will be the layered product QE team's repositories.
 
 ## Test Execution
+
 >This is the responsibility of the PQE team.
 
 Similar to the layered product deployment the code being used to setup and execute the tests will be coming from the product QE's test repositories. If there are tests failing for any reason the product QE team will need to fix the problem in their test repo.
 
 ## Triggering Cadence
+
 >This is the responsibility of the CSPI-QE team.
 
 If there is ever a change to the cadence of testing we must update the cron configuration for all scenarios. The current model is to trigger using the latest OpenShift build on a weekly cadence each Monday morning.
 
 See the [Triggering Guide](../../OCP_CI_Tutorials/Triggering/Triggering_Guide.md) for more information.
 
 ## Reporting Tests
+
 >This is the responsibility of the CSPI-QE team.
 
 This is part of the value that the CSPI-QE team provides. We will be responsible for consolidating reports for many layered product scenarios consumable by PM, leadership, engineering & QE. It will be the responsibility of the CSPI-QE to ensure that the test results being generated by the layered products tests are reported quickly & accurately to all the appropriate places.
```

Comment:
```suggestion
This is part of the value that the CSPI-QE team provides. We will be responsible for consolidating reports for many layered product scenarios consumable by PM, leadership, engineering & QE. It will be the responsibility of the CSPI-QE team to ensure that the test results being generated by the layered products tests are reported quickly & accurately to all the appropriate places.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)